server/routerlicious/packages/services-utils/src/winstonLumberjackEngine.ts

```diff
@@ -15,12 +15,12 @@ export class WinstonLumberjackEngine implements ILumberjackEngine {
         const obj = {
             eventName: lumber.eventName,
             id: lumber.id,
-            properties: propObj,
+            properties: JSON.stringify(propObj),
             type: LumberType[lumber.type],
             timestamp: new Date(lumber.timestamp).toISOString(),
             durationInMs: lumber.durationInMs,
             successful: lumber.successful,
-            exception: lumber.exception,
+            exception: lumber.exception?.message,
```

Comment:
Previously this would have included the name and stackTrace, right? Is it intentional to limit to just the message?

Reply:
Actually, using `lumber.exception` and giving it to Winston was printing an empty object. That's why I tried to extract the message. Let me look up how to properly "stringify" an error object.